ee/codegen/src/types/vellum.ts

```diff
@@ -810,6 +810,7 @@ export interface ExecutionCounterWorkflowReference {
 }
 
 export interface DictionaryWorkflowReferenceEntry {
+  id?: string;
```

Comment:
[nit] Let's leave a comment about how this is here temporarily to support prompt input_variables for now, and that in the future, we won't need to have prompt attributes reference each other like this once we clean up prompt inputs from the node